app/scripts/modules/core/delivery/executionGroup/executionGroups.component.ts

```diff
@@ -0,0 +1,42 @@
+import {IComponentController, IComponentOptions, module} from 'angular';
+
+import {EXECUTION_GROUP_COMPONENT} from './executionGroup.component';
+import {Application} from 'core/application/application.model';
+
+import './executionGroups.less';
+
+export class ExecutionGroupsController implements IComponentController {
+  public application: Application;
+
+  public groups: any;
+  public sortFilter: any;
+
+  static get $inject() { return ['$state', 'ExecutionFilterModel']; }
+
+  constructor(private $state: angular.ui.IStateService,
```

Comment:
This can be imported directly from ui-router ala `import {IStateService} from 'angular-ui-router';`